lib/models/src/main/java/org/sagebionetworks/repo/web/TwoFactorAuthEnabledRequiredException.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.repo.web;
+
+public class TwoFactorAuthEnabledRequiredException extends RuntimeException {
```

Comment:
I wold throw the same exception we throw when attempting to download a file that requires 2fa.  The UI does not need a special exception.